### PR TITLE
FOUR-14931 Task > Quick Fill: menus and headers options are blocked b…

### PR DESCRIPTION
**Pull request by gproly**
https://processmaker.atlassian.net/browse/FOUR-14931

1. Create a Request
2. Open the Task
3. Press Quick Fill button
4. Wait for the table
5. Select one row in the table
6. Press over eye icon in the table

ci:next

### CODE REVIEW

File: resources/js/tasks/components/QuickFillPreview.vue [hunk 1]
```diff
@@ -277,16 +274,21 @@ export default {
...
-      if(!searchBar){
-        return;
-      }
-      let obj = searchBar.classList;
-      if (sw === true) {
-        obj.add("global-search-bar-z-index");
-      } else {
-        obj.remove("global-search-bar-z-index");
+      let names = [".global-search-bar", ".navbar-nav"];
+      for (let i = 0; i < names.length; i++) {
```
Comment (gustavobascope) [inline resources/js/tasks/components/QuickFillPreview.vue:278]: SonarQube is making the observation to use for (const name of names) instead of simple for

**Review by gustavobascope — COMMENTED** [review]
SonarQube is making the observation to use for (const name of names) instead of simple for

**Review by gustavobascope — APPROVED** [review]
approved

**Comment by caleeli** [comment]
@gproly 

I added `ci:next` to the PR description

cc @nolanpro @gustavobascope

**Pull request merged**